gutenberg.php

```diff
@@ -9,6 +9,8 @@
  * @package gutenberg
  */
 
+define( 'GUTENBERG__PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
```

Comment:
Ah, I didn't notice that, my bad.

In a new PR, I'll fix those things.

Thanks!